res/controllers/Numark-Mixtrack-Pro-FX-scripts.js

```diff
@@ -113,11 +113,11 @@ MixtrackProFX.shutdown = function() {
     midi.sendSysexMsg(shutdownSysex, shutdownSysex.length);
 };
 
-MixtrackProFX.EffectUnit = function(unitNumber) {
-    this.unitNumber = unitNumber;
-    this.group = "[EffectRack1_EffectUnit" + unitNumber + "]";
+MixtrackProFX.EffectUnit = function(deckNumber) {
+    this.deckNumber = deckNumber;
+    this.group = "[EffectRack1_EffectUnit" + deckNumber + "]";
```

Comment:
This change doesn't make sense without changing this to controlling the QuickEffects.

Reply:
As we discussed, we'll change it to QuickEffects after effect chain preset saving/loading is available in Mixxx.